### ISSUE TITLE
Cannot read property 'path' of undefined 

### ISSUE THREAD

**Issue by jwright04** [samselikoff/ember-cli-mirage#668]
just installed `ember-cli-mirage` and I'm receiving the following error when serving my application:

```
TypeError: Cannot read property 'path' of undefined
    at Class.module.exports.treeForAddon (~directory/test/test-app/node_modules/ember-cli-mirage/node_modules/ember-lodash/index.js:11:74)
    at Class._treeFor (~directory/test/test-app/node_modules/ember-cli/lib/models/addon.js:322:31)
    at Class.treeFor (~directory/test/test-app/node_modules/ember-cli/lib/models/addon.js:290:19)
    at ~directory/test/test-app/node_modules/ember-cli/lib/models/addon.js:244:32
    at Array.map (native)
    at Class.eachAddonInvoke (~directory/test/test-app/node_modules/ember-cli/lib/models/addon.js:242:22)
    at Class.treeFor (~directory/test/test-app/node_modules/ember-cli/lib/models/addon.js:289:20)
    at Class.module.exports.treeFor (~directory/test/test-app/node_modules/ember-cli-mirage/index.js:32:34)
    at ~directory/test/test-app/node_modules/ember-cli/lib/broccoli/ember-app.js:463:20
    at Array.map (native)
```

The error goes away when I use version `0.1.11`

**Comment by daidi** [comment]
same problem  ..
how to solve?

**Comment by visoft** [comment]
Yep, same issue here. Goes away with v0.1.11

**Comment by samselikoff** [comment]
Sorry, conf wifi ran out after a bad publish. Use 0.1.1-1 for now

**Comment by samselikoff** [comment]
Should be fixed with [0.1.13](https://github.com/samselikoff/ember-cli-mirage/releases/tag/v0.1.13)
